src/services/order_watcher_service.ts

```diff
@@ -23,7 +24,11 @@ export class OrderWatcherService {
         // in the future we can attempt to retry these a few times. Ultimately if we
         // cannot validate the order we cannot keep the order around
         // Validate the local state and notify the order watcher of any missed orders
-        const { accepted, rejected } = await meshUtils.addOrdersToMeshAsync(this._meshClient, signedOrders);
+        const { accepted, rejected } = await meshUtils.addOrdersToMeshAsync(
+            this._meshHttpEndpoint,
+            this._meshClient,
```

Comment:
It would cleaner if there was a `meshClient` class defined in this codebase that under-the-hood decided which transport to use, instead of passing around both an `HTTPEndpoint` and a `WSClient` everywhere. Otherwise, we don't have a clean separation of concerns.